src/index.test.ts

```diff
@@ -1,32 +1,20 @@
-import {describe, expect, test} from '@jest/globals'
-import fetchMock from "jest-fetch-mock"
+import { describe, expect, test } from '@jest/globals'
+import axios from "axios"
+import MockAdapter from "axios-mock-adapter"
```

Comment:
I would rather use `Nock` for mocking requests. 
https://www.npmjs.com/package/nock

With nock it's easier to test requests to **real** server, check headers, check body and query strings and their API pretty simple.

Reply:
Changed